ansible_events/builtin.py

```diff
@@ -17,6 +17,11 @@
 import yaml
 from durable import lang
```

Comment:
@evacchi We don't need this line.
Can you run flake8 . on your repo
You will get all the linter issues.
You might have to tweak your local tox.ini so it excludes the venv directory 
I have a PR for that https://github.com/benthomasson/ansible-events/pull/129